### PR TITLE
Automatic detect address is local or remote.

### PR DESCRIPTION
**Pull request by saeed-golshan**
Because of backward compatibility i didn't remove isLocal from constructor but i  removed every isLocal parameter from example and check  example worked properly. I made a prediction for windows local file address (third logical express line 348).

### CODE REVIEW

File: lib/audioplayers.dart [hunk 1]
```diff
@@ -338,14 +338,14 @@ class AudioPlayer {
   /// If [isLocal] is false, [url] must be a remote URL.
   Future<int> play(
     String url, {
-    bool isLocal = false,
+    bool isLocal,
```
Comment (luanpotter) [inline lib/audioplayers.dart]: should we add a default value of null so that we don't need to specify it?

Comment (luanpotter) [reply lib/audioplayers.dart]: just realized it's default value is null by default :)

**Comment by luanpotter** [comment]
@erickzanardo want to take a look? you can merge right after if it LGTY

**Comment by erickzanardo** [comment]
Only thing missing is the CHANGELOG, other than that, looks good to me.

**Comment by luanpotter** [comment]
@erickzanardo I will merge and add the entry myself, good catch

**Pull request merged**